### PR TITLE
add examples of scaffolder action inputs to docs

### PR DESCRIPTION
**Pull request by punkle**
Signed-off-by: Brian Fletcher <brian@roadie.io>

## Hey, I just made a Pull Request!

add examples of scaffolder action inputs to docs

Here is what it looks like:

image

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/scaffolder-backend/src/scaffolder/actions/builtin/catalog/register.ts [hunk 1]
```diff
@@ -34,9 +56,10 @@ export function createCatalogRegisterAction(options: {
     | { catalogInfoUrl: string; optional?: boolean }
     | { repoContentsUrl: string; catalogInfoPath?: string; optional?: boolean }
   >({
-    id: 'catalog:register',
+    id,
     description:
       'Registers entities from a catalog descriptor file in the workspace into the software catalog.',
+    examples,
```
Comment (punkle) [inline plugins/scaffolder-backend/src/scaffolder/actions/builtin/catalog/register.ts:62]: Ive only added three examples to start with. But if it is considered a good approach, I can complete some more examples.

File: plugins/scaffolder/src/components/ActionsPage/ActionsPage.tsx [hunk 2]
```diff
@@ -67,6 +73,26 @@ const useStyles = makeStyles(theme => ({
...
+    title: 'Description',
+    field: 'description',
+    width: '20%',
+  },
+  {
+    title: 'Example',
+    render: ({ example }) => {
+      return (
+        <Grid>
+          <SyntaxHighlighter showLineNumbers style={yaml}>
```
Comment (punkle) [inline plugins/scaffolder/src/components/ActionsPage/ActionsPage.tsx:87]: Im not convinced about the styling.

**Comment by punkle** [comment]
@Sarabadu thank you. I'll try that.

**Comment by punkle** [comment]
@Sarabadu it didnt work 😢

**Comment by Sarabadu** [comment]
@punkle I checked the branch, tried on my local and seems to work for me, not sure what else could be causing this...

another try could be:
- delete dist-types
- run `yarn tsc`
- run `yarn build:api-reports:only`

**Pull request closed without merging**